other/postgres-entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [[ "${DB_BACKUP}" == "true" ]]; then
```

Comment:
plelase add default true value in the current .env file

Reply:
also  document in the wiki both new variables

Reply:
its already set to true